src/Plate/index.stories.tsx

```diff
@@ -18,20 +18,20 @@ export default {
 
 const data: Array<PlateWell> = [
   {
-    coordinates: { row: COORDINATES_ROWS[0], column: COORDINATES_COLUMNS[6] },
+    coordinates: { row: COORDINATES_ROWS[0]!, column: COORDINATES_COLUMNS[6]! },
```

Comment:
We could provide a  `function coordinateRow(index: Coordinates['column']): Coordinates['row']`  (and one for columns) within `constants.ts`. That way the redundancy of bang operators will be removed.

```suggestion
    coordinates: { row: coordinateRow(0), column: coordinateColumn(6) },
```